examples/bpi-monolith-test-suite/bpi/components/workgroup/workgroup.ts

```diff
@@ -0,0 +1,35 @@
+import { BpiSubject } from "../identity/bpiSubject";
+import { Workstep } from "./workstep";
```

Comment:
see my previous comment on relationship between workgroup workflow and worksteps @ognjenkurtic